internal/tui/model_list.go

```diff
@@ -21,10 +24,23 @@ import (
 )
 
 var (
-	appStyle            = lipgloss.NewStyle().Padding(1, 2)
-	listItemSidePadding = 0
-	listItemStyle       = lipgloss.NewStyle().PaddingLeft(listItemSidePadding).Bold(true)
-	listItemHelpStyle   = lipgloss.NewStyle().Align(lipgloss.Right)
+	appStyle  = lipgloss.NewStyle().Margin(1, 2)
+	helpStyle = lipgloss.NewStyle().
+			PaddingLeft(2).
+			Foreground(lipgloss.Color("241"))
+	inputStyle = lipgloss.NewStyle().
+			MarginLeft(1).
+			MarginBottom(2).
+			BorderStyle(lipgloss.NormalBorder()).
+			BorderForeground(lipgloss.Color("63")).
+			Foreground(lipgloss.Color("#87448b"))
+	titleStyle = lipgloss.NewStyle().
+			Foreground(lipgloss.Color("#FAFAFA")).
+			Background(lipgloss.Color("#5d5dd2")).
```

Comment:
Long-term we might want to think about standardizing these styles in some way. Something to think about if we continue to develop around the TUI.

Reply:
I agreed on this.